### PR TITLE
[Feature](function) support array_contains_all function

### PR DESCRIPTION
**Pull request by zhangstar333**
## Proposed changes
docs link:
https://github.com/apache/doris-website/pull/914

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

File: be/src/vec/functions/array/function_array_contains_all.cpp [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+                assert_cast<const DataTypeArray&>(*right_data_type).get_nested_type());
+        DCHECK(left_nested_type->equals(*right_nested_type))
+                << "data type " << arguments[0]->get_name() << " not equal with "
+                << arguments[1]->get_name();
+        return std::make_shared<DataTypeUInt8>();
+    }
+
+    Status execute_impl(FunctionContext* context, Block& block, const ColumnNumbers& arguments,
+                        size_t result, size_t input_rows_count) const override {
+        auto left_column =
```
Comment (HappenLee) [inline be/src/vec/functions/array/function_array_contains_all.cpp:63]: consider how we opt the const column

File: regression-test/suites/nereids_p0/sql_functions/array_functions/test_array_contains_all.groovy [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            DISTRIBUTED BY HASH(`k1`) BUCKETS 10
+            PROPERTIES (
+            "replication_allocation" = "tag.location.default: 1",
+            "storage_format" = "V2"
+            )
+        """
+    sql """
+        CREATE TABLE ${tableName2} (
+        id bigint not null ,
+        array_col1   Array<BigInt>,
```
Comment (HappenLee) [inline regression-test/suites/nereids_p0/sql_functions/array_functions/test_array_contains_all.groovy:40]: better more test on array<decimal> or array<string>

**Comment by zhangstar333** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by BiteTheDDDDt — APPROVED** [review]

**Pull request merged**